src/tcp/stream.rs

```diff
@@ -12,6 +14,12 @@ use tokio_io;
 const RENDEZVOUS_TIMEOUT_SEC: u64 = 10;
 const RENDEZVOUS_INFO_EXCHANGE_TIMEOUT_SEC: u64 = 120;
 
+/// Final connection handshake message.
+/// One peer reads incoming stream and waits for this message, while the other sends this
+/// message indicating wish to connect.
+type ChooseConnectionMsg = [u8; 6];
+const CHOOSE: ChooseConnectionMsg = [b'c', b'h', b'o', b'o', b's', b'e'];
```

Comment:
Why are we going for such convolutions ? Can we not have a simpler way something like:
```
const BLAH: &[u8] = b"Don't care what the length of this is";
```
and then use it as:
```
fn bar(b: &[u8]) { ... }
fn foo() { /* ... */  bar(BLAH);  /* ... */ }
```
That way you don't have to keep changing/calculating lengths when you change your string constants. Plus it's more manageable and readable and less verbose than specifying characters separately.

Reply:
not really sure why, I simply extracted this fragment from other function, but I'll change it since I agree with you